### PR TITLE
WDP200802-2

### PR DESCRIPTION
**Pull request by mARC3L87**
Przy wyświetlaniu większej liczby np. 123 element źle się zachowuje. Poprawić tak, aby pozwalał na wyświetlanie liczb z zakresu 0-99999 (max. 5 cyfr). 
- dodano lewy padding i zmieniono justify-content na flex-start

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,72 @@
+[0805/192827.442:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0805/192827.448:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0805/192827.448:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
Comment (Varenthein) [inline debug.log:3]: We don't need this file at all.

File: src/components/common/ProductBox/ProductBox.module.scss [hunk 2]
```diff
@@ -75,4 +77,14 @@
     justify-content: space-between;
     align-items: center;
   }
+
+  &:hover{
+    .buttons{
+      opacity: 1;
+    }
+
```
Comment (Varenthein) [inline src/components/common/ProductBox/ProductBox.module.scss:85]: I see that you have in this PR commits from the older task. I assume, that you've started from older task branch. Never do that. Always start a new branch for a new task from `master`.

Comment (mARC3L87) [reply src/components/common/ProductBox/ProductBox.module.scss:85]: Actually I did that. What should I do with that? Create completely new branch form master and start over?

Comment (Varenthein) [reply src/components/common/ProductBox/ProductBox.module.scss:85]: That is the "safest" option. Could you do that?

I'm closing this PR then.

Comment (Varenthein) [reply src/components/common/ProductBox/ProductBox.module.scss:85]: Use the name WDP20080202-2v2 for this new branch that you'll make.

**Pull request closed without merging**